Makefile

```diff
@@ -63,3 +63,11 @@ image/push:
 .PHONY: image/build/push
 image/build/push: image/build image/push
 
+.PHONY: vendor/check
+vendor/check: vendor/fix
+	git status | grep 'nothing to commit, working tree clean'
```

Comment:
Using a slightly different approach here than the format checks. The output from `go mod vendor` is the same in both cases where there are changes and there are not changes so didn't provide a good way to fail out using diff.

Reply:
Maybe we should explicitly check the vendor directory status?

```
git status vendor/
```

It might also be worth considering changing it to use git diff like the format check to avoid grepping a message that might change over time:

```
git diff --exit-code vendor/
```

Reply:
Yes, I like this. I had a concern about the message changing also. I'll implement it that way.